hr_employee_lastnames/models/hr_employee.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+from odoo.addons.hr_employee_firstname.models.hr_employee import UPDATE_PARTNER_FIELDS
+
+UPDATE_PARTNER_FIELDS += ['lastname2']
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    lastname2 = fields.Char("Second last name")
+
+    @api.model
+    def _get_name_lastnames(self, lastname, firstname, lastname2=None):
+        return self.env['res.partner']._get_computed_name(
+            lastname, firstname, lastname2)
+
+    def _build_name_in_create(self, vals):
+        if any([field in vals for field in ['firstname', 'lastname', 'lastname2']]):
```

Comment:
`if any(field...`

Remove the `[` in order to used it as iterator instead of list

It will faster.

Reply:
done